jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env roseus
+;; vim: set ft=lisp:
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l")
+
+
+(defmethod jsk_2016_01_baxter_apc::baxter-interface
+  (:graspingp (arm) t)
+  )
+
+
+(jsk_2016_01_baxter_apc::baxter-init)
+(setq *irtviewer* (make-irtviewer))
+(send *irtviewer* :change-background #f(0.8 0.8 0.8))
+
+(setq right_gripper_bodies nil)
```

Comment:
I'm changing it. But using vim, I don't have the completion about variable name when I use `-`, how do you fix it?